### PR TITLE
Timer

### PR DESCRIPTION
**Pull request by vycius**

### CODE REVIEW

**Comment by kuprelis** [comment]
:+1:

**Comment by ghost** [comment]
:+1: nu k

File: app/src/main/java/com/nmakademija/nmaakademija/entity/TimeUntilSession.java [hunk 1]
```diff
@@ -1,29 +1,38 @@
...
+    private long sessionEnd;
+
+    private SimpleDateFormat dateFormat;
 
-    private final DateFormat formatter = new SimpleDateFormat("MM-dd HH:mm:ss", Locale.US);
 
     public TimeUntilSession(long StartTime, long EndTime) {
-        _SessionStart = StartTime;
-        _SessionEnd = EndTime;
+        sessionStart = StartTime * 1000;
```
Comment (Dekedro) [inline app/src/main/java/com/nmakademija/nmaakademija/entity/TimeUntilSession.java]: 1000 shouldn't be hardcoded, but declared as 'final static' variable.

File: app/src/main/java/com/nmakademija/nmaakademija/entity/TimeUntilSession.java [hunk 2]
```diff
@@ -1,29 +1,38 @@
...
 
-    public String returnTime() {
+    public String returnTime(Context context) {
         long now = new Date().getTime();
 
-        long timeLeft = Math.max(_SessionStart * 1000 - now, _SessionEnd * 1000 - now);
-
-        String dateFormatted = formatter.format(timeLeft);
+        long timeLeft = Math.max(sessionStart - now, sessionEnd - now);
+        long daysLeft = timeLeft / (24 * 60 * 60 * 1000);
```
Comment (Dekedro) [inline app/src/main/java/com/nmakademija/nmaakademija/entity/TimeUntilSession.java]: These values also should be 'final static'

**Pull request merged**